Makefile

```diff
@@ -12,11 +12,19 @@ export GO111MODULE = on
 ifeq ($(OS),Windows_NT)
        # Just assume we're in interactive mode on Windows
        INTERACTIVE = 1
-       VERSION = $(shell type version.txt)
+			 VERSION = $(shell type version.txt)
 else
        INTERACTIVE := $(shell [ -t 0 ] && echo 1)
 endif
 
+ifdef GO_SPACEMESH_VERSION
```

Comment:
I'd rather avoid introducing another variable here. Why not just pass this in as `VERSION=...`?